From 5c849e861d6c969ce31bfc60011f202d687d0bfd Mon Sep 17 00:00:00 2001
From: Arpit Agarwal <arp@apache.org>
Date: Mon, 7 Mar 2016 12:19:53 -0800
Subject: [PATCH 1804/2848] HDFS-9906. Remove spammy log spew when a datanode
 is restarted. (Contributed by Brahma Reddy
 Battula)

(cherry picked from commit 02949b2151f1eecce3e1bcdbbec1afac164a6065)
(cherry picked from commit aca4310f326966c0b0dd3b41d9cc0ecbe34bc7e3)

Change-Id: I085f83d713f3bd863a8ab921dd5a7ebb965d7021
---
 .../hdfs/server/blockmanagement/BlockManager.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
index 2768513..960431b 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
@@ -2650,7 +2650,7 @@ private Block addStoredBlock(final BlockInfo block,
       corruptReplicas.removeFromCorruptReplicasMap(block, node,
           Reason.GENSTAMP_MISMATCH);
       curReplicaDelta = 0;
-      blockLog.warn("BLOCK* addStoredBlock: Redundant addStoredBlock request"
+      blockLog.debug("BLOCK* addStoredBlock: Redundant addStoredBlock request"
               + " received for {} on node {} size {}", storedBlock, node,
           storedBlock.getNumBytes());
     }
-- 
1.7.9.5

